utils/utils.hpp

```diff
@@ -213,5 +264,35 @@ class Utils
 
     return 0;
   }
+
+  /**
+   * Fills a vector with paths to all files in directory.
+   *
+   * @param path Path to Directory.
+   * @param pathVector A vector of type filesystem::path, which will be filled
+   *                   paths for all files / folders in given directory path.
+   * @param absolutePath Boolean to determine if path is absolute or relative.
+   */
+  static void ListDir(const std::string& path,
+                      std::vector<boost::filesystem::path>& pathVector,
+                      const bool absolutePath = false)
```

Comment:
Can you let me know the use case of this function?

Reply:
This fills the pathVector with paths of all files in the `path`. For example, if we have a folder called images with 30 files with names 1.jpeg, 2.jpeg and so on then pathVector will be equal to `{"1.jpeg", "2.jpeg" ...}`. I use it to get all annotations and images in the folder for PASCAL VOC dataloader.

Reply:
Aha, thanks for letting me know! nice :)